dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -0,0 +1,304 @@
+  echo -e "\e[93;1mFile service.yaml is missing.\e[0m"
+  rm service.yaml
+fi
+cd ../ || exit 1
+
+# GET CHE-APP CONFIGS
+if ! (curl -L0fs https://raw.githubusercontent.com/redhat-developer/rh-che/$RH_CHE_GITHUB_BRANCH/openshift/rh-che.app.yaml -o rh-che.app.yaml 2>1 > /dev/null); then
+  echo -e "\e[91;1mCould not download che app definition config!\e[0m"
+  exit 1
+fi
+if ! (curl -L0fs https://raw.githubusercontent.com/redhat-developer/rh-che/$RH_CHE_GITHUB_BRANCH/openshift/rh-che.config.yaml -o rh-che.config.yaml 2>1 > /dev/null); then
+  echo -e "\e[91;1mCould not download che config yaml!\e[0m"
+  exit 1
+fi
+
+chmod 777 wait_until_postgres_is_available.sh
+chmod 777 deploy_postgres_only.sh
+echo -e "\e[92;1mGetting deployment scripts done.\e[0m"
+
+# DEPLOY POSTRES
+if [ "$RH_CHE_WIPE_SQL" == "true" ]; then
+  if (oc get dc postgres 2>1 > /dev/null); then
+    oc delete dc postgres
+  fi
+  deployPostgres
+else
+  if ! (oc get dc postgres 2>1 > /dev/null); then
+    deployPostgres
+  fi
+fi
+
+# APPLY CHE CONFIGMAP
+#yq '' ./rh-che.config.yaml | cat
+CHE_CONFIG_YAML=$(yq ".\"data\".\"che-workspace-che-server-endpoint\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/wsmaster/api\" | 
+                     .\"data\".\"che-host\" = \"che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io\" | 
+                     .\"data\".\"infra-bootstrapper-binary-url\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/agent-binaries/linux_amd64/bootstrapper/bootstrapper\" | 
+                     .\"data\".\"che-api\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/api/\" | 
+                     .\"data\".\"che-websocket-endpoint\" = \"wss://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/api/websocket\" | 
+                     .\"data\".\"che-keycloak-auth-server-url\" = \"https://sso.prod-preview.openshift.io/auth\" | 
+                     .\"data\".\"keycloak-github-endpoint\" = \"https://auth.prod-preview.openshift.io/api/token?for=https://github.com\" | 
+                     .\"data\".\"keycloak-oso-endpoint\" = \"https://sso.prod-preview.openshift.io/auth/realms/fabric8/broker/openshift-v3/token\" | 
+                     .\"data\".\"service.account.secret\" = \"\" | 
+                     .\"data\".\"service.account.id\" = \"\" | 
+                     .\"data\".\"che.jdbc.username\" = \"$RH_CHE_JDBC_USERNAME\" | 
+                     .\"data\".\"che.jdbc.password\" = \"$RH_CHE_JDBC_PASSWORD\" | 
+                     .\"data\".\"che.jdbc.url\" = \"$RH_CHE_JDBC_URL\" | 
+                     .\"metadata\".\"name\" = \"che\" " ./rh-che.config.yaml)
+
+#echo "$CHE_CONFIG_YAML"
+if ! (echo "$CHE_CONFIG_YAML" | oc apply -f - 2>1 > /dev/null); then
+  echo -e "\e[91;1mFailed to apply configmap.\e[0m"
+  exit 1
+fi
+echo -e "\e[92;1mChe config deployed on \e[34m$RH_CHE_PROJECT_ID\e[0m"
+
+# PROCESS CHE APP CONFIG
+sed -i 's/rhche/che/g' ./rh-che.app.yaml
```

Comment:
Why do you need to do this replacement? Isn't this massive replacement of `rhche` error prone?

Reply:
It is only used as a `name:` property in the config files. I can redo it to leave it as rhche, but that would require modifications in other places including the keycloak redirects provided by Ilja because the server url automatically changes from `https://che.whatever...` to `https://rhche.whatever...` and I didn't find a different way to change that.

Maybe there only needs to be a slight modification for that to work? however I'm not sure what I need to change as the origin deployment starts throwing out issues like `app "rhche" not found` if I don't modify the entire config.

I think you are correct on this call, will have to look into it.

Reply:
done and fixed to use default route from config